react-components/src/data-providers/core-dm-provider/check3dConnectedEquipmentQuery.ts

```diff
@@ -144,6 +67,54 @@ function getRevisionsPointCloudVolumes(object3dTableName: string): QueryTableExp
   };
 }
 
+type Image360ConnectionExpressions<Prefix extends string> = Record<
+  `${Prefix}_360_annotation_edges` | `${Prefix}_360_image_nodes`,
+  QueryTableExpressionV3
+>;
+
+function getRevisionsImage360Annotations<const Prefix extends string>(
+  prefix: Prefix,
+  object3dTableName: string,
+  revisionRefs: DmsUniqueIdentifier[]
+): Image360ConnectionExpressions<Prefix> {
+  const edgesResultExpression = `${prefix}_360_annotation_edges`;
+  const imagesResultExpession = `${prefix}_360_image_nodes`;
+
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Remove?

Reply:
Typescript is unable to infer that the strings `edgesResultExpression` and `imagesResultExpressions` are in fact the keys of `Image360ConnectionExpressions<Prefix>`. I have even tried annotating the them explicitly with the type (e.g. ``const edgesResultExpression: `${Prefix}_360_annotation_edges` = `${prefix}_360_annotation_edges`;``), but they're still not recognized. Furthermore, the eslint rule `consistent-type-assertions` require us to use `const v: T = { ... }` rather than `const v = { ... } as T`, which makes sense most of the time, but not in this case where Typescript does not see that the value is in fact assignable to the type at hand.

Reply:
(This code was eventually removed, so the above comment is irrelevant)